tools/visual-pipeline-and-platform-evaluation-tool/app.py

```diff
@@ -631,6 +748,20 @@ def on_run(
                     outputs=[run_button],
                 )
 
+
+                benchmark_button.click(
+                on_benchmark,
+                inputs=[
+                    recording_channels,
+                    inferencing_channels,
+                    fps_floor,
+                    object_detection_model,
+                    object_detection_device,
+                    input_video_player,
+                ],
+                outputs=[best_config_textbox],
+                )

```

Comment:
Indent this section properly.
Remove the recording and inference channels, you don't use them for benchmarking (right?)